neurodsp/filt/utils.py

```diff
@@ -149,11 +152,15 @@ def compute_transition_band(f_db, db, low=-20, high=-3):
         The lower limit that defines the transition band, in dB.
     high : float, optional, default: -3
         The upper limit that defines the transition band, in dB.
+    return_freqs : bool, optional, default: False
+        Whether to return a tuple of (lower, upper) frequency bounds for the transition band.
 
     Returns
     -------
     transition_band : float
         The transition bandwidth of the filter.
+    f_range : tuple of (float, float), optional, default: False
```

Comment:
I don't think this should have `, optional, default: False`
A second line description (on 164) could / should say: "Only returned if `return_freqs` is True."